src/notifications/notification.ts

```diff
@@ -61,4 +61,11 @@ export default class Notification implements Comparable<Notification> {
 
     return mdStr;
   }
+
+  /**
+   *  Debug method that makes Notification objects printable inside templated strings
+   */
+  public toString(): string {
+    return this.title.text + this.content;
```

Comment:
We should probably add a linebreak or something between the title and the content

Reply:
Added a dash to have single-line logs